### ISSUE TITLE
[FR] Add option to resize referenced grid/board/calendar views

### ISSUE THREAD

**Issue by rampa3** [AppFlowy-IO/AppFlowy#4696]
### Description

I would like to request ability to resize referenced grid/board/calendar views, to allow the block to take up more or less space on the page depending on user's needs. This would allow users to shrink the block if they don't need to see as many contents visible as by default and want to conserve page space, while on the other hand allow users who for example want to see as many rows of a grid as possible to do so. 

### Impact

In my opinion, this feature would be mostly generally useful for everyone, but particurarly for users that might want to fit more views onto screen without the need to scroll, or on the other side for example for students, so they can use views of databases in their notes and view them trough reference without having to scroll the view itself after reaching it in the notes.

### Additional Context

This feature request came from my own personal use of AppFlowy for management of my lecture notes, seminar notes, assignments and other university related data, where I use mainly grids in my lecture notes to construct tables, and would prefer to be able to enlarge the view to see the whole grid when referencing the notes at later date than having to scroll it separately from the page itself.

**Comment by LucasXu0** [comment]
@rampa3 Do you mean that you want to resize the **height** of the referenced database view on the document page? Currently, the height of the referenced database view is fixed.

**Comment by rampa3** [comment]
@LucasXu0 Yes, that would be my feature request. I think it could be useful function.

**Comment by annieappflowy** [comment]
@rampa3 , we have made improvements around inline views. Please give it a try!
